### ISSUE TITLE
mirrord binary missing in vscode extension in WSL 

### ISSUE THREAD

**Issue by t4lz** [metalbear-co/mirrord#1101]
### Bug Description

User can't use mirrord vscode extension on windows+WSL, because the mirrord binary is missing inside WSL (because the universal version is being installed instead of the linux one, probably).

### Steps to Reproduce

1. Install mirrord version 3.27 on vscode on windows.
2. Try to use mirrord.
3. Fail.

### Backtrace

```shell
ENOENT: no such file or directory, chmod '/home/someusername/.vscode-server-insiders/extensions/metalbear.mirrord-3.27.0/mirrord'
```

### Your operating system and version

Windows

### Local process

Any

**Comment by t4lz** [comment]
If the problem is indeed that the universal version is installed instead of the linux-x64 one, a stupid and easy solution would be to include the linux-x64 binary in the universal version. Obviously it would be better if we could figure out how to make sure the linux version is installed in WSL.

**Comment by aviramha** [comment]
> If the problem is indeed that the universal version is installed instead of the linux-x64 one, a stupid and easy solution would be to include the linux-x64 binary in the universal version. Obviously it would be better if we could figure out how to make sure the linux version is installed in WSL.

If it installs the universal on the Linux I fear that it might do it for other platforms as well, like when you'd download for macOS it might also try to use universal for some reason.
I think the solution is to make it like IntelliJ where we have all versions inside and we choose at runtime..

**Comment by t4lz** [comment]
> have all versions inside and we choose at runtime

Yesterday I saw some bits of documentation that are relevant for that solution so including them here:
On [how to dynamically pick compiled code based on architecture](https://code.visualstudio.com/api/advanced-topics/remote-extensions#supporting-nonx8664-hosts-or-alpine-linux-containers):
> If you are dynamically acquiring compiled code, you can add support by detecting non-x86_64 targets using `process.arch` and downloading versions compiled for the right architecture. If you are including binaries for all supported architectures inside your extension instead, you can use this logic to use the correct one.

On [how to dynamically pick compiled code based on if the extension is running in WSL](https://code.visualstudio.com/api/advanced-topics/remote-extensions#using-native-node.js-modules):
> 1. Include (or dynamically acquire) both sets of binaries [...].
> 2. Check to see if `vscode.extensions.getExtension('your.extensionId').extensionKind === vscode.ExtensionKind.Workspace` to set up the correct binaries based on whether the extension is running remotely or locally.

**Comment by aviramha** [comment]
Solved with #1106
